src/main/scala/de/zalando/beard/renderer/ForContextFactory.scala

```diff
@@ -16,13 +16,17 @@ object ForContextFactory {
     newContext(forIterationContext.templateIteratorIdentifier) match {
       case map: Map[String, Any] => {
         val index = forIterationContext.currentIndex
-        val newMap = map.updated("isLast", index == forIterationContext.collectionOfContexts.size - 1)
-          .updated("isFirst", index == 0)
-          .updated("isOdd", index % 2 == 1)
-          .updated("isEven", index % 2 == 0)
+        val last = index == forIterationContext.collectionOfContexts.size - 1
+        val newMap = map.
+          updated("isLast", last).
+          updated("isNotLast", !last).
```

Comment:
yeah... I also needed this... I thought we'll implement a another small grammar for ifs, using parser combinators... maybe for later :)